extensions/wikia/FeedsAndPosts/js/feedsAndPosts.js

```diff
@@ -5,10 +5,16 @@ require(['jquery'], function ($) {
 
 	$('#WikiaRail').one('afterLoad.rail', function() {
 		$.getScript(fpLibrary, function () {
+			var wikiName = $('meta[property="og:site_name"]').prop('content');
```

Comment:
🥇

Reply:
Wouldn't `mw.config.get('wgSiteName')` (or just `wgSiteName`, but that's deprecated in newer MediaWiki versions) be more consistent, though? Not like it matters much now that the pull request has been merged, but I feel like creativity isn't the point here.